docs/how-to-guides/software-management/software-support.md

```diff
@@ -49,4 +72,15 @@ Prioritization meetings are typically a loosely guided discussion in which the t
 1. Ask for any closing remarks
 
 ??? Note "Issue deferral"
-    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
\ No newline at end of file
+    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization. 
+
+
+## Continuous Support 
+
+Internal developers have time to perform basic support tasks throughout the week. We refer to this as continuous support, as it's support time between dedicated sprints. This is to stay on top of GitHub notifications. 
+
+As a user, you should expect feedback on an issue within a week of posting. If urgent, you can ping `@DOI-USGS/astrogeology-developers`. If the GitHub issue is open with no activity for a long enough time it will become stale and eventually auto-closed. Often, issues are not ignored, but are saved for a support sprint. 
```

Comment:
This seems to be contradictory. 

1. You should expect feedback within a week. If you need it sooner, ping this group. ✅ 
2. If the issue is open for a long time with no activity it will become stale. ❓ 

If that were in two different paragraphs, not under a continuous support heading, sure. But as it is organized, it reads like one continuous thought. You should expect feedback and if there is not activity your issue will auto close.

Suggest removing the inactivity stuff entirely or moving it to a completely new section.